### PR TITLE
update gr.m3u

### PR DESCRIPTION
**Pull request by georgoskar**
#EXTINF:0 tvg-id="ext" group-title="channels",baraza hd greek music hits
https://eco.streams.ovh:443/BarazaTV/BarazaTV/playlist.m3u8
#EXTINF:0 tvg-id="ext" group-title="channels",baraza tv classic 80s-90s  music
https://eco.streams.ovh:443/BarazaClassic/BarazaClassic/playlist.m3u8
#EXTINF:0 tvg-id="ext" group-title="channels",baraza tv music  hits HD  
https://5cc309e93b721.streamlock.net:443/8452/8452/playlist.m3u8

### CODE REVIEW

File: channels/gr.m3u [hunk 1]
```diff
@@ -12,9 +12,11 @@ http://176.9.123.140:1935/atticatv/atticatv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Greek" tvg-logo="https://goo.gl/mNqHoo" group-title="",Axeloos
 http://srv.viiideo.gr:1935/axeloos/live/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Greek" tvg-logo="" group-title="Music",Baraza TV Classic Hits
-https://greekyoutube.eu:444/live/5e300ade5dadc/index.m3u8
+https://eco.streams.ovh:443/BarazaClassic/BarazaClassic/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Greek" tvg-logo="" group-title="Music",Baraza TV Greek Music Hits
 https://eco.streams.ovh/BarazaTV/BarazarazaTV/BarazaTV/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Greek" tvg-logo="" group-title="Music"Baraza music hits
```
Comment (freearhey) [inline channels/gr.m3u]: Lacks a comma before the name of the channel.

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by georgoskar** [comment]
#EXTINF:0 tvg-id="ext" group-title="channels",baraza hd greek music hits
https://eco.streams.ovh:443/BarazaTV/BarazaTV/playlist.m3u8
#EXTINF:0 tvg-id="ext" group-title="channels",baraza tv classic 80s-90s
 music
https://eco.streams.ovh:443/BarazaClassic/BarazaClassic/playlist.m3u8
#EXTINF:0 tvg-id="ext" group-title="channels",baraza tv music  hits HD
https://5cc309e93b721.streamlock.net:443/8452/8452/playlist.m3u8

Στις Τρί, 3 Μαρ 2020 στις 10:48 π.μ., ο/η Aleksandr Statciuk <
notifications@github.com> έγραψε:

**Comment by georgoskar** [comment]
#EXTINF:0 tvg-id="ext" group-title="channels",baraza hd greek music hits
https://eco.streams.ovh:443/BarazaTV/BarazaTV/playlist.m3u8
#EXTINF:0 tvg-id="ext" group-title="channels",baraza tv classic 80s-90s  music
https://eco.streams.ovh:443/BarazaClassic/BarazaClassic/playlist.m3u8
#EXTINF:0 tvg-id="ext" group-title="channels",baraza tv music  hits HD  
https://5cc309e93b721.streamlock.net:443/8452/8452/playlist.m3u8

**Comment by freearhey** [comment]
You need to add the missing comma so I can approve this PR.

**Comment by georgoskar** [comment]
#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Greek" tvg-logo="
https://i.imgur.com/mW6c2QR.png " group-title="Music",Baraza TV Classic Hits


https://eco.streams.ovh:443/BarazaClassic/BarazaClassic/playlist.m3u8


#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Greek" tvg-logo="
https://i.imgur.com/mW6c2QR.png " group-title="Music",Baraza TV Greek Music
Hits


https://eco.streams.ovh/BarazaTV/BarazarazaTV/BarazaTV/playlist.m3u8


#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Greek" tvg-logo="
https://i.imgur.com/mW6c2QR.png " group-title="Music",Baraza TV Music Dance
Hits


https://5cc309e93b721.streamlock.net:443/8452/8452/playlist.m3u8

Στις Τετ, 4 Μαρ 2020 στις 1:48 π.μ., ο/η Aleksandr Statciuk <
notifications@github.com> έγραψε:

**Comment by georgoskar** [comment]
sorry. I hadn't seen it.thanks

Στις Τετ, 4 Μαρ 2020 στις 10:01 π.μ., ο/η barazaradio makis galinos <
tsirlokopio@gmail.com> έγραψε:

**Comment by freearhey** [comment]
As I wrote above, you need to add the missing comma and update your pull request. If you don't know how to do it, watch this video: https://www.youtube.com/watch?v=e41HPOHX9aE.

**Review by freearhey — APPROVED** [review]

**Comment by georgoskar** [comment]
link correction link baraza greek music hits missing the door 443.
#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Greek" tvg-logo="
https://i.imgur.com/mW6c2QR.png" group-title="Music",Baraza TV Greek Music
Hits https://eco.streams.ovh/BarazaTV/BarazarazaTV/BarazaTV/playlist.m3u8

#EXTINF:0 tvg-id="ext" group-title="channels",baraza hd greek music hits
https://eco.streams.ovh:443/BarazaTV/BarazaTV/playlist.m3u8

Στις Τετ, 4 Μαρ 2020 στις 12:25 μ.μ., ο/η Aleksandr Statciuk <
notifications@github.com> έγραψε:

**Comment by georgoskar** [comment]
Please view this email in an HTML compatible email client.

**Pull request merged**